docs/rest-api/source/includes/wp-api-v3/deposits.md

```diff
@@ -481,7 +413,7 @@ Fetches a deposit by ID.
 
 If a deposit is found for the provided ID, the response will return a [**Deposit**](#deposit-object) object.
 
-If no deposit is found for the provided ID, the response will be an empty array.
+If no deposit is found for the provided ID, the response will return a `500` status code.
```

Comment:
NB: The HTTP response code was changed to `404` on the server, the client still responds with `500`.

Reply:
Interesting … I like the idea that we work towards no 500 errors from any of our APIs.

Reply:
Yeh this came up in #4342 which implemented the server returning `404`, only for client to swap it to a `500`.

For now, the docs should reflect the current state of the REST API but we can open a new issue to ensure `404`'s are passed through to client REST API responses.

Reply:
Nice, yea I'm thinking long term here. Also might be a cool way into more monitoring of our APIs – e.g. monitoring can tell us when we return 500, we can work toward no 500s (maybe in API paths that we own).